pages/index.js

```diff
@@ -29,12 +34,20 @@ export default function Home() {
       </Head>
 
       <Main>
-        <div>
-          {cardData.map((card) => (
-            <Card key={card.id} name={card.name} />
-          ))}
-        </div>
-        <CreateNewForm onAddNewData={appendCard} />
+        <ul>
+          <li>
+            {cardData.map((card) => {
+              return (
+                <Card
+                  key={card.id}
+                  name={card.name}
+                  onDelete={() => deleteCard(card.id)}
+                />
+              );
+            })}
+          </li>
```

Comment:
This is not what I meant. Every card is a li element inside the ul. Please change it again. (instead of using an 'article' for the card, use a li instead in Card.js)

Reply:
@F-Kirchhoff  i have made the changes i hope i got you this time right